.editorconfig

```diff
@@ -0,0 +1,9 @@
+[*]
+indent_style = tab
+indent_size = 2
```

Comment:
Just for consistency, we typically use a tab size of 4

Reply:
I set this to 2 since this repo is almost entirely JSON files, and a size of 2 makes it more readable with the nesting - would you still like me to change it? 

Personally I feel like this is a special case

Reply:
Leaving it as 2 for the JSON is fine. Can we add an additional section for the scripts though, like you have for the yaml?

Reply:
Sure - I'll make a specific exemption for the JSON